docs/how-tos.md

```diff
@@ -210,13 +210,36 @@ Custom relationships are useful if the relationship data is not stored in a buil
 
 ### Metadata
 
-cozy-client will also automatically insert and update the standard document metadata if you provide a `cozyMetadata` object to the schema.
+cozy-client will also automatically insert and update the standard document metadata. You have to provide some info when you initialize the client:
```

Comment:
`metadata` being an addition, it's better if we can leave it as optional. "have to" -> "may"